docs/github-action.md

```diff
@@ -11,7 +11,7 @@ nav_order: 6
 
 OSV-Scanner is offered as a GitHub Action. We currently have two different GitHub Actions:
 
-1. An action that performs a vulnerability scan on a [regular schedule](./github-action.md#scheduled-scans).
+1. An action that performs a single vulnerability scan, which can be configured to scan on a [regular schedule](./github-action.md#scheduled-scans), or used as a check [on releases](./github-action.md#scan-on-release) to prevent releasing with known vulnerabilities in dependencies.
```

Comment:
It isn't good practice that we have the scheduled scan and the release scan grouped together in this first item in the list and then they aren't grouped that way in the instructions below which are in this order:
1. Scheduled scan
2. PR scan
3. Release scan

Would you prefer to update the order of the instructions, or how they are grouped in lines 14 & 15?

Reply:
That makes sense, let's change it to this order so that scheduled and release is together:

1. PR scan
2. Scheduled scan
3. Release scan

Should we still separate out the two entries on line 14&15? We'll probably have to reword it to something like 
> We currently support 3 different ways to use Github Actions:
>
> 1. PR Scan ...
> 2. Scheduled Scan ...
> 3. Release Scan ...
>

Reply:
Updated the ordering in f62f6c3